### ISSUE TITLE
z-index bug with skip to content link

### ISSUE THREAD

**Issue by graeme-lindsay** [bbc/simorgh#5864]
**Description**
When the skip to content link has focus, the bottom of this is being clipped off the NAV on Burmese (also seen on some other services). 
This is possibly owing to the script height. 

<img width="1394" alt="Screen Shot 2020-03-04 at 14 50 46" src="https://user-images.githubusercontent.com/22519682/76842820-ca4e3d00-6832-11ea-9d06-16248a5f5cc3.png">

**To Recreate**
Go to https://www.bbc.com/burmese
Tab to the skip to content link
Note that the bottom of link has been clipped off. 

**Proposed solution**
Should not be cut off and the borders should be visible see: 
<img width="1394" alt="Screen Shot 2020-03-04 at 14 50 46" src="https://user-images.githubusercontent.com/22519682/76843305-80b22200-6833-11ea-98e9-c3e70f5d844c.png">

**Comment by graeme-lindsay** [comment]
@aeroplanejane is going to follow up with Catherine

**Comment by aeroplanejane** [comment]
Noted by @greenc05 in this thread: https://bbcnews.slack.com/archives/CJT0284G2/p1583333534265000
@gavinspence What do you want to do with this one?

**Comment by paruchurisilpa** [comment]
Checked that the Skip to content link appears completely without the bottom line not appearing..Checked for burmese and persian. LGTM..
